README.adoc

```diff
@@ -473,6 +479,34 @@ doc:2:test_obvious_matching_with_assert_not_matches()
 	Running test_obvious_notmatching_with_assert_not_matches ... SUCCESS
 ```
 
+=== *assert_within_delta*
+
+    assert_within_delta <expected num> <actual num> <max delta> [message]
+
+Asserts that the expected num matches the actual num up to a given max delta.
+Given an expectation of 5 and a delta of 2 this would match 3, 4, 5, 6, and 7:
```

Comment:
I think that _either_ the description should specifically call out that decimal numbers are not supported OR the feature be modified to support them. Right now it says 

```
 actual value [3.4] is not a number
```

Obviously supporting decimals would be preferable but then you get into weird floating point edge cases if someone dares to specify a delta that's also a decimal.

Reply:
Bash does not support decimal numbers so I don't plan to. But you're right, this must be explicit in the doc.